### DISCUSSION TITLE
issues installing the-future branch

### DISCUSSION THREAD

**Discussion by HosAkh** [asmvik/yabai#1031] [Q&A]
I had yabai installed but running issues on Monterey so I tired to install the future branch. I followed all the instructions here: https://github.com/koekeishiya/yabai/wiki/Installing-yabai-(from-HEAD)#updating-to-latest-head

but still couldn't tell if I was on the future branch and having issues. So I uninstalled yabai and started from top of the listed page above but once I get to the command line: 
```
brew install koekeishiya/formulae/yabai --HEAD
codesign -fs 'yabai-cert' $(which yabai)
```

I get this error:
```
Error: Your Command Line Tools (CLT) does not support macOS 12.
It is either outdated or was modified.
Please update your Command Line Tools (CLT) or delete it if no updates are available.
```

I have done all 3 methods of update (even manually by going to the site and installing command line for Xcode 13.0) but still getting the same error. 

What am doing wrong? 

I've been trying to get yabai working for 2 days and been hitting a wall. Thank you

**Comment by Menchen** [comment]
Try deleting old CLT and install again
```
sudo rm -rf /Library/Developer/CommandLineTools
sudo xcode-select --install
```
Also I believe that Xcode from App store is required.

**Comment by joshtwo** [comment]
I don't know if you're still trying to use that branch, but currently the master branch is more up-to-date.

**Reply by HosAkh** [reply]
> I don't know if you're still trying to use that branch, but currently the master branch is more up-to-date.

I installed the master branch and added the config file with demo code and the auto script but nothing happens. I restarted yabai and even my computer and still nothing
